events/views.py

```diff
@@ -219,3 +225,20 @@ def stats(request):
 
     return render_to_response(
         'stats/index.html', data, context_instance=RequestContext(request))
+
+def events_category(request, id, slug):
+    """
+        Filter evets for category
+    """
+    events = CategoryEvent.objects.filter(category_id=id)
+
+    ct = Category.objects.get(id=id)
+    category = ct.name
+
+    data = {
+        'events': events,
+        'category': category,
```

Comment:
Why you pass just the category name?

If you pass the category object then in the template you can access all values easily category.name category.slug...